spec/models/schools_admin_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
This file is using 4 spaces instead of 2 in some places. Also, please add a blank line at the end of the file.

Reply:
will do